code/game/objects/item.dm

```diff
@@ -97,6 +97,15 @@
 	var/unbreakable = FALSE                    // Whether or not this weapon degrades.
 	var/anomaly_shielding					   // 0..1 value of how well it shields against xenoarch anomalies
 
+	///Used when thrown into a mob
+	var/mob_throw_hit_sound
```

Comment:
Will have to say, this one is a bit odd
Hitting a fleshbag of a mob would sound different than hitting a stone golem, or a metallic robot.